### PR TITLE
update the tilt doc with screenshots

### PR DESCRIPTION
**Pull request by sonasingh46**
/kind documentation

**What this PR does / why we need it**:
This PR adds tilt screenshot to deploy flavor workload cluster so as to give more clarity for the first time contributors.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1618

**Special notes for your reviewer**:

**TODOs**:

- [x] squashed commits
- [x] includes documentation
- [x] adds unit tests

**Release note**:

```release-note
None
```

### CODE REVIEW

File: templates/flavors/README.md [hunk 1]
```diff
@@ -36,7 +33,13 @@ Please note your tilt-settings.json must contain at minimum the following fields
     }
 }
 ```
+After having the tilt-settings.json, follow these two steps to deploy a workload cluster
```
Comment (mboersma) [inline templates/flavors/README.md:36]:
```suggestion
After updating tilt-settings.json, follow these two steps to deploy a workload cluster:
```

File: templates/flavors/README.md [hunk 2]
```diff
@@ -36,7 +33,13 @@ Please note your tilt-settings.json must contain at minimum the following fields
...
 
+1. ``tilt up`` in the root of cluster-api-provider-azure repo. Note that the tilt-settings.json also resides here in the
```
Comment (mboersma) [inline templates/flavors/README.md:38]:
```suggestion
1. Run ``make tilt-up`` in the root of cluster-api-provider-azure repo. Note that the tilt-settings.json also resides here in the 
```

File: templates/flavors/README.md [hunk 3]
```diff
@@ -36,7 +33,13 @@ Please note your tilt-settings.json must contain at minimum the following fields
     }
 }
 ```
+After having the tilt-settings.json, follow these two steps to deploy a workload cluster
 
+1. ``tilt up`` in the root of cluster-api-provider-azure repo. Note that the tilt-settings.json also resides here in the 
+root of this repo. After this, press `space` to open the tilt UI in browser. See the following example:
```
Comment (mboersma) [inline templates/flavors/README.md:39]:
```suggestion
root of this repo. After tilt has initialized, press `space` to open the tilt web UI in a browser. See the following example:
```

File: docs/book/src/developers/development.md [hunk 4]
```diff
@@ -217,9 +215,10 @@ The cluster-api management components that are deployed are configured at the `/
 
 ⚠️ Note that when developing with `tilt` as described above, some `clusterctl` commands won't work. Specifically, `clusterctl config` and `clusterctl generate` may fail. These commands expect specific releases of CAPI and CAPZ to be installed, but the `tilt` environment dynamically updates and installs these components from your local code. `clusterctl get kubeconfig` will still work, however.
 
-After your kind management cluster is up and running with Tilt, you can [configure workload cluster settings](#customizing-the-cluster-deployment) and deploy a workload cluster by opening the `tilt` web UI and clicking the clockwise arrow icon ⟳ on a resource listed, such as "aks-aad," "ipv6," or "windows."
+After your kind management cluster is up and running with Tilt, you can deploy a workload cluster by opening the `tilt` web UI and clicking the clockwise arrow icon ⟳ on a resource listed, such as "aks-aad," "ipv6," or "windows."
+Deploying a workload cluster from Tilt UI is also termed as flavor cluster deployment. Please refer [here](../../../../templates/flavors/README.md#Running-flavor-clusters-as-a-tilt-resource) to learn more about this.
```
Comment (mboersma) [inline docs/book/src/developers/development.md:219]:
```suggestion
Please refer to [Running flavor clusters as a tilt resource](../../../../templates/flavors/README.md#Running-flavor-clusters-as-a-tilt-resource) to learn more about this.
```

**Review by mboersma — APPROVED** [review]
/lgtm

**Comment by CecileRobertMichon** [comment]
@Jont828 would be great to get your review on this since you went through the process of setting up tilt for the first time recently

**Comment by Jont828** [comment]
Looks good, having the expected output and showing the arrow icon will definitely help! /lgtm

**Pull request merged**